tests/install/run_case.sh

```
@@ -0,0 +1,134 @@
+#                                  silently kill the whole script instead of no-op'ing the
+#                                  way an unmatched if/elif branch used to. This mode fails
+#                                  (via `set -e`, no "still alive" line) if that regresses.
+
+set -u
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+
+export afc_path="$REPO_ROOT"
+
+# shellcheck source=/dev/null
+source "$REPO_ROOT/include/units/registry.sh"
+source "$REPO_ROOT/include/constants.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./include/constants.sh was not specified as input (see shellcheck -x).

(SC1091)